base/applications/rapps/lang/es-ES.rc

```diff
@@ -214,9 +214,9 @@ BEGIN
     IDS_SELECTEDFORINST "Seleccionados para instalar"
     IDS_MISMATCH_CERT_INFO "El certificado que usa es desconocido:\nSujeto: %s\nEmisor: %s\n¿Quiere continuar a pesar de ello?"
     IDS_UNABLE_PATH "Formato de ruta incorrecto."
-    IDS_APP_DISPLAY_DETAILS "details"
-    IDS_APP_DISPLAY_LIST "list"
-    IDS_APP_DISPLAY_TILE "tile"
+    IDS_APP_DISPLAY_DETAILS "detalles"
+    IDS_APP_DISPLAY_LIST "listado"
```

Comment:
Yo pondría "lista", pero para gustos.
Es el combobox que te permite seleccionar opciones de visualización tipo "iconos grandes, iconos pequeños, detalles, lista, etc" en el RAPPS.

Reply:
vale, lo tengo corregido en mi copia. No sabía lo que era realmente. Si hubiera sido un listbox sí hubiera sido apropiado.

Reply:
Ah yes, since you're at it... Could you please use capital letter for the 1st one of each of these words? (for the IDS_APP_DISPLAY_xxx)

Reply:
> Ah yes, since you're at it... Could you please use capital letter for the 1st one of each of these words? (for the IDS_APP_DISPLAY_xxx) (I'm going to adjust that in master)

Done!